sensum-pwa/src/app/App.res

```diff
@@ -1,44 +1,132 @@
-
 @react.component
 let make = () => {
-  
-  Sensations.getSensationsLength()->Promise.then(index => {
+  Sensations.getSensationsLength()
+  ->Promise.then(index => {
     Js.Console.log(index)
     Promise.resolve()
-  })->Promise.catch( err => {
+  })
+  ->Promise.catch(err => {
     Js.Console.log(err)
     Promise.resolve()
-  })->ignore
-  
+  })
+  ->ignore
+
   // Try with this instead, and check the COMPILE time error:
   // Sensations.getSensationByIndex(~index=1000)->Promise.then(sensation => {
-  Sensations.getSensationByIntIndex(~index=1000)->Promise.then(sensation => {
+  Sensations.getSensationByIntIndex(~index=1000)
+  ->Promise.then(sensation => {
     Js.Console.log(sensation)
     Promise.resolve()
-  })->Promise.catch( err => {
+  })
+  ->Promise.catch(err => {
     Js.Console.error(err)
     Promise.resolve()
-  })->ignore
-  
+  })
+  ->ignore
+
   let s: Sensations.sensation = {
     author: "mk",
-    message: "los Granujas sean unidos porque esa es la ley 1ra"
+    message: "los Granujas sean unidos porque esa es la ley 1ra",
   }
-  
-  Sensations.newSensation(~s=s)->Promise.then(confirmedTx => {
+
+  Sensations.newSensation(~s)
+  ->Promise.then(confirmedTx => {
     Js.Console.log(confirmedTx)
     Sensations.getLatestSensation()
   })
   ->Promise.then(sensation => {
     Js.Console.log(sensation)
     Promise.resolve()
-  })->Promise.catch( err => {
+  })
+  ->Promise.catch(err => {
     Js.Console.log(err)
     Promise.resolve()
-  })->ignore
-  
-  <div>
-    <h1 className="text-4xl font-bold"> {"Sensum PWA"->React.string} </h1>
-    <p className="text-2xl"> {"viteJs + react + rescript + tailwindcss"->React.string} </p>
+  })
+  ->ignore
+
+  <div className="bg-black w-full h-full">
```

Comment:
i just learn that if you wrap your app in `flex flex-col h-screen overflow-hidden` you don't need the `fixed` class in the header. Just for curiosity 😆 

```
  <div class="flex flex-col h-screen overflow-hidden">
    <!-- Navbar -->
    <header class="w-full bg-purple-900 border-purple-600 border-b-2 flex justify-between">
      <h1 class="text-4xl text-purple-50">
        Sensum
      </h1>
      {component}
    </header>
    <!-- Navbar end -->

    <main class="flex-1 overflow-y-scroll">
      <div>
        <p>element</p>
      </div>
      <!-- ... -->
    </main>
</div>
```

Reply:
nice!